oasis/models/iforest.py

```diff
@@ -132,74 +124,47 @@ def run(self):
         for key in self.job.metrics:
             if key in Metrics:
                 val = Metrics[key]
-                config = {"features": ["mean", "std"]}
-                if key in self.cfg.metrics and "features" in self.cfg.metrics[key]:
-                    config = self.cfg.metrics[key]
+                if key not in self.cfg.metrics:
+                    continue
 
-                t = Thread(target=self.run_action, args=(key, val, config, ))
+                t = Thread(target=self.run_action,
```

Comment:
Can we also get rid of `Thread` by using facilities from tornado?

May be done in another PR, if you find this is difficult.

Reply:
ok, I will fix this problem in single pr